packages/terra-form-fieldset/src/terra-dev-site/doc/example/FieldsetExamples.jsx

```diff
@@ -46,18 +46,18 @@ class FieldsetExamples extends React.Component {
           required
         >
           <Field label="First" isInline required htmlFor="first">
```

Comment:
Let's use the same label as the label in the field. In this case, the label would be `First`